src/webhook/verify.ts

```diff
@@ -1,10 +0,0 @@
-import nacl from "tweetnacl";
-
-// validate signature using public key
-export function verify(msg: Buffer, sig: string, publicKey: string) {
```

Comment:
I don't think we should be removing these functions & tests

Reply:
Will bring in the tests from substreams-sink-webhook.